src/dialogxml/widgets/button.hpp

```diff
@@ -74,9 +74,12 @@ class cButton : public cControl {
 	cButton(cDialog& parent,eControlType t);
 private:
 	bool manageFormat(eFormat prop, bool set, boost::any* val) override;
+	void defaultTextSize();
 	std::string fromList;
 	static rectangle btnRects[13][2];
 protected:
+	/// Size of the button's descriptive text
+	short textSize = 0;
```

Comment:
I don't think 0 is a good default here.

Reply:
It doesn't stay 0, the 0 indicates that defaultTextSize() should do its thing.

Reply:
I'm not sure if that actually makes sense though…? It would make sense if defaultTextSize ran _after_ parsing, but that's not what I see here.

Reply:
Oh! I misunderstood the call order. Yeah, it doesn't need default 0.